app/services/store_participant_eligibility.rb

```diff
@@ -42,6 +42,8 @@ def send_ineligible_notification_email
       mailer_name = "#{participant_profile.participant_type}_#{@participant_eligibility.reason}_email"
       if IneligibleParticipantMailer.respond_to? mailer_name
         IneligibleParticipantMailer.send(mailer_name, **{ induction_tutor_email: induction_tutor.email, participant_profile: participant_profile }).deliver_later
+      elsif mailer_name == "mentor_previous_participation_email"
```

Comment:
I'd be tempted to short circuit it before the `If IneligibleParticipantMailer.respond_to?` line i.e.

```ruby
next if mailer_name == "mentor_previous_participation_email"

If IneligibileParticipantMailer.respond_to? mailer_name
...
```